### PR TITLE
Automatically parse projects from pages and update list dynamically

### PR DESCRIPTION
**Pull request by QEDK**
This sets the base for pull-request support. The bot does the following:
* Checks if the project page is updated (run as cronjob?), if yes then - (implemented)
* Create a `parsebot` branch with the updated template, then - (implemented)
* Create a pull request with the commit(s), then - (implemented) (see #35 for e.g.)
* Update the branch until the pull request is closed, then - (implemented) (see #38)
* Delete the branch. (see #38)

~Note: The test fails because of linting, should be fixed once I'm done with the complete idea.~ Fixed!

### CODE REVIEW

File: bots/parsebot.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
```
Comment (xSavitar) [inline bots/parsebot.py:1]: We can order the import statements to be in alphabetical order of some sort (ASC for example) and let the from statements be below the strict imports.

The idea is just code structure as the code base grows, we can have strategies to quickly navigate it.

File: bots/parsebot.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import html2text
+import requests
+import shlex
+import json
+import time
+import re
+import os
+
+
+def scan(r):
```
Comment (xSavitar) [inline bots/parsebot.py:14]: Could `r` be something more descriptive?

File: bots/parsebot.py (hunk 1 above)
Comment (QEDK) [reply bots/parsebot.py:1]: Resolved in latest commit.

File: bots/parsebot.py (hunk 2 above)
Comment (QEDK) [reply bots/parsebot.py]: Renamed to `session` in latest commit.

**Pull request merged**